### PR TITLE
Merge v4.x.x -> v5.x.x

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Please review carefully; there were quite a few non-trivial conflicts.

File: src/swarm/util/Verify.d [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
```
Comment (gavin-norman-sociomantic) [inline src/swarm/util/Verify.d:1]: Ahh, we'll have to keep removing this module whenever we merge :/ Not sure if there's a better way.

Comment (nemanja-boric-sociomantic) [reply src/swarm/util/Verify.d:1]: I would just leave it there and remove it in v6.x.x.

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Comment by nemanja-boric-sociomantic** [comment]
Feel free to push, can't rebase.

**Pull request merged**